hudi-hadoop-common/src/main/java/org/apache/hudi/io/hadoop/HoodieAvroParquetReader.java

```diff
@@ -68,7 +68,7 @@ public HoodieAvroParquetReader(HoodieStorage storage, StoragePath path) {
     // by the Reader (for proper config propagation to Parquet components)
     this.storage = storage.newInstance(path, tryOverrideDefaultConfigs(storage.getConf().newInstance()));
     this.path = path;
-    this.parquetUtils = FileFormatUtils.getInstance(HoodieFileFormat.PARQUET);
+    this.parquetUtils = new HoodieHadoopIOFactory(storage).getFileFormatUtils(HoodieFileFormat.PARQUET);
```

Comment:
same here, can we use `HoodieIOFactory` instead of specific impl of HoodieIOFactory?

Reply:
Fixed.